tests/unit/test_connect_url.py

```diff
@@ -5,15 +5,41 @@
 import time
 from repo2docker.app import Repo2Docker
 
+
+# Minimal Dockerfile to make build as fast as possible
+DOCKER_FILE = """
+FROM python:3.7-slim
+# install the notebook package
+RUN pip install --no-cache --upgrade pip && \
+    pip install --no-cache notebook
+
+# create user with a home directory
+ARG NB_USER
+ARG NB_UID
+ENV USER ${NB_USER}
+ENV HOME /home/${NB_USER}
+
+RUN adduser --disabled-password \
+    --gecos "Default user" \
+    --uid ${NB_UID} \
+    ${NB_USER}
+WORKDIR ${HOME}
+USER ${NB_USER}
```

Comment:
This is the only difference to [Min's original minimal Dockerfile](https://github.com/binder-examples/minimal-dockerfile/blob/master/Dockerfile). Without it the notebook complains that it is being executed as root.